portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,53 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,400i,500,700|Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Kristin Aoki</h1>
+      <img src="/images/last_page-9.png" alt="Image of Kristin Aoki" style="width:30%;display:block;margin-left:auto;
```

Comment:
Consider moving styles to a .css file and using classes to include the styles in your markup. Inline styles are fine but keeping all the CSS in one place might be a better organization option. Your call on this one.